.github/ISSUE_TEMPLATE/iceberg_bug_report.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: Iceberg Bug report 🐞
+description: Problems, bugs and issues with Apache Iceberg
+labels: ["kind:bug"]
+body:
+  - type: dropdown
+    attributes:
+      label: Apache Iceberg version
+      description: >
+        What Apache Iceberg version are you using?
+      multiple: false
+      options:
+        - "0.13.1 (latest released)"
+        - "0.13.0"
+        - "0.12.1"
+        - "0.12.0"
+        - "0.11.1"
+        - "0.11.0"
+        - "main (development)"
+    validations:
+      required: true
```

Comment:
Do we need to make this required? I'd probably start with optional and then see whether that's a problem. If people fill it out most of the time then that's better than nothing.

Reply:
Agree, made it optional.